src/styles/variable.css

```diff
@@ -1,10 +1,10 @@
 :root {
   /* Font Sizes */
-  --font-size-sm: 14px;
-  --font-size-md: 18px;
-  --font-size-lg: 36px;
-  --size-xl: 40px;
-  --font-size-2xl: 48px;
+  --font-size-sm: 0.875rem; /* 14px */
+  --font-size-md: 1.125rem; /* 18px */
+  --font-size-lg: 2.25rem; /* 36px */
+  --size-xl: 2.5rem; /* 40px */
```

Comment:
Is the `--size-xl` not have correct naming?

Reply:
Fixed